TWLight/emails/templates/emails/access_code_email-body-html.html

```diff
@@ -0,0 +1,9 @@
+<html>
+<body>
+  <p>Your approved application to {{ partner }} has now been finalised. Your access code or login details can be found below:</p>
+
+  <p><b>{{ access_code }}</b></p>
+
+  <p>{{ body }}</p>

```

Comment:
I feel like this should probably be `access_code_instructions` instead of `body` for legibility.
```suggestion
  <p>{{ access_code_instructions }}</p>
```